testfile1.py

```
@@ -27,8 +27,8 @@ def main():
         return
 
     try:
-        num1 = float(input("Enter first number: "))
-        num2 = float(input("Enter second number: "))
+        num1 = float(intup("Enter first number: "))
```

Ruff (0.11.9)

Undefined name `intup`

(F821)